[debian-8]: curl.h not found, despite libcurl-dev being installed

```
/home/vagrant/workspace/CouchDB-Matrix/OS/Debian-8/src/couch/priv/couch_js/http.c:85:23: fatal error: curl/curl.h: No such file or directory
 #include <curl/curl.h>
```

--- a/debian-7/Vagrantfile
+++ b/debian-7/Vagrantfile
@@ -8,7 +8,7 @@ Vagrant.configure(2) do |config|
   config.vm.provision "shell", inline: <<-SHELL
     sudo sh -c 'echo deb http://http.debian.net/debian wheezy-backports main > /etc/apt/sources.list.d/wheezy-backports.list'
     sudo apt-get -qq update
-    sudo apt-get install -y git emacs23-nox autoconf autoconf-archive libtool build-essential nodejs-legacy texlive-full python-sphinx openjdk-7-jre-headless shunit2 libmozjs185-dev libicu-dev erlang/wheezy-backports
+    sudo apt-get install -y git emacs23-nox autoconf autoconf-archive libtool build-essential nodejs-legacy texlive-full python-sphinx openjdk-7-jre-headless shunit2 libmozjs185-dev libicu-dev libcurl4-openssl-dev erlang/wheezy-backports
     sudo sh -c 'curl https://www.npmjs.com/install.sh | sh'
   SHELL
 end

--- a/debian-8/Vagrantfile
+++ b/debian-8/Vagrantfile
@@ -6,7 +6,7 @@ Vagrant.configure(2) do |config|
   end
 
   config.vm.provision "shell", inline: <<-SHELL
-    sudo apt-get install -y git emacs-nox autoconf autoconf-archive libtool build-essential nodejs npm texlive-full python-sphinx openjdk-7-jre-headless shunit2 libmozjs185-dev libicu-dev erlang
+    sudo apt-get install -y git emacs-nox autoconf autoconf-archive libtool build-essential nodejs npm texlive-full python-sphinx openjdk-7-jre-headless shunit2 libmozjs185-dev libicu-dev libcurl4-openssl-dev erlang
     sudo npm install -g npm
   SHELL
 end

